common/display/virtualdisplay.cpp

```diff
@@ -60,6 +94,32 @@ VirtualDisplay::~VirtualDisplay() {
 
   resource_manager_->PurgeBuffer();
   compositor_.Reset();
+#ifdef HYPER_DMABUF_SHARING
+  if (mHyperDmaBuf_Fd > 0 && display_index_ == 0) {
+    auto it = mHyperDmaExportedBuffers.begin();
+    for (; it != mHyperDmaExportedBuffers.end(); ++it) {
+      struct ioctl_hyper_dmabuf_unexport msg;
+      int ret;
+      msg.hid = it->second.hyper_dmabuf_id;
+      // Todo: hyper dmabuf free delay is fixed to 1s now!
```

Comment:
the code here unexport the hyper dmabuf reference IOTG's code, they set the delay time consider the system  stability, 1000 is safety time, in theory it can decrease. I will check if they have evaluated a less number.

Reply:
Maybe change this to something like: " TODO: find a reduced dmabuf free delay time"